deepbox/src/main/java/ch/cyberduck/core/deepbox/DeepboxListService.java

```diff
@@ -152,4 +178,19 @@ else if(thirdLevelId.endsWith(TRASH)) {
         }
         return list;
     }
+
+    // list by modifiedTime desc to keep only most recent with the same name
+    private void listChunk(final Path directory, final NodeContent inbox, final AttributedList<Path> list, final Set<String> closed) throws ApiException {
```

Comment:
Can be simplified if server returns result with `modifiedTime desc` to set duplicate flag in attributes if node with the same name exists in list.

Reply:
Instead removing entries from the list we can just mark them as duplicates.